src/behaviors/utils.js

```diff
@@ -0,0 +1,56 @@
+// @flow
```

Comment:
These functions seem to be specific to the `toggle` behavior, so they would better be located under `src/behavior/toggle` directory. If some are generic enough to be reused somewhere else, we can turned into services under `src/services`

Reply:
These are generic, I plan on using them to implement `hide` and `show` actions. As we migrate other behaviors to `src/behaviors`, it will use them as well.

Do you recommend I move it to `src/services/behaviors/index.js`?

Reply:
yes would be a good place if they are shared among behaviors and not used by any other part of the code